frontend/components/vault/VaultLiquidationTransactionFlow.vue

```diff
@@ -146,6 +142,16 @@ export default Vue.extend({
         wasLiquidated(): boolean {
             return this.vaultTransaction.state === 'liquidated';
         },
+        liquidationLimits(): LiquidationLimits {
+            return {
```

Comment:
I think it's possible to pass a type which has more properties than needed. So you can pass the whole `VaultTransaction` to the prop that asks for `LiquidationLimits`, no need to hand-pick certain keys